### ISSUE TITLE
评估结果在哪里可以看到

### ISSUE THREAD

**Issue by xiaowuzilou** [PaddlePaddle/PaddleSeg#273]
“模型训练完成，使用下述命令启动评估
python pdseg/eval.py --use_gpu --cfg ./configs/deeplabv3p_xception65_optic.yaml”
这个评估结果也没有发现生成的图片或文件，屏幕上也没有发现。请问在哪里可以看到？

**Comment by LutaoChu** [comment]
评估结果会显示在命令行，包括miou、macc、kappa等等指标。不会生成图片或文件

**Comment by xiaowuzilou** [comment]
您好！
&nbsp; 感谢您的解答回复。不过，我在“评估操作命令行”出现的界面上仔细检查过了，没有发现您说的评估结果。具体见《运行界面.docx》.
此外，在用visualdl查看训练结果时，出现"gbk”错误。如《错误界面.docx》中所示。
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; 期待您的解答！


------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"LutaoChu"<notifications@github.com&gt;;
发送时间:&nbsp;2020年6月1日(星期一) 上午10:21
收件人:&nbsp;"PaddlePaddle/PaddleSeg"<PaddleSeg@noreply.github.com&gt;;
抄送:&nbsp;"1345933573"<1345933573@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [PaddlePaddle/PaddleSeg] 评估结果在哪里可以看到 (#273)





 
评估结果会显示在命令行 ，包括miou、macc、kappa等等指标。不会生成图片或文件

**Comment by xiaowuzilou** [comment]
(base) E:\PaddleSeg-develop>python pdseg/eval.py --use_gpu --cfg ./configs/unet_optic.yaml
{'AUG': {'AUG_METHOD': 'unpadding',
         'FIX_RESIZE_SIZE': (512, 512),
         'FLIP': False,
         'FLIP_RATIO': 0.5,
         'INF_RESIZE_VALUE': 500,
         'MAX_RESIZE_VALUE': 600,
         'MAX_SCALE_FACTOR': 2.0,
         'MIN_RESIZE_VALUE': 400,
         'MIN_SCALE_FACTOR': 0.5,
         'MIRROR': True,
         'RICH_CROP': {'ASPECT_RATIO': 0.33,
                       'BLUR': False,
                       'BLUR_RATIO': 0.1,
                       'BRIGHTNESS_JITTER_RATIO': 0.5,
                       'CONTRAST_JITTER_RATIO': 0.5,
                       'ENABLE': False,
                       'MAX_ROTATION': 15,
                       'MIN_AREA_RATIO': 0.5,
                       'SATURATION_JITTER_RATIO': 0.5},
         'SCALE_STEP_SIZE': 0.25},
 'BATCH_SIZE': 4,
 'DATALOADER': {'BUF_SIZE': 256, 'NUM_WORKERS': 8},
 'DATASET': {'DATA_DIM': 3,
             'DATA_DIR': './dataset/optic_disc_seg/',
             'IGNORE_INDEX': 255,
             'IMAGE_TYPE': 'rgb',
             'NUM_CLASSES': 2,
             'PADDING_VALUE': [127.5, 127.5, 127.5],
             'SEPARATOR': ' ',
             'TEST_FILE_LIST': './dataset/optic_disc_seg/test_list.txt',
             'TEST_TOTAL_IMAGES': 180,
             'TRAIN_FILE_LIST': './dataset/optic_disc_seg/train_list.txt',
             'TRAIN_TOTAL_IMAGES': 900,
             'VAL_FILE_LIST': './dataset/optic_disc_seg/val_list.txt',
             'VAL_TOTAL_IMAGES': 180,
             'VIS_FILE_LIST': './dataset/optic_disc_seg/test_list.txt'},
 'EVAL_CROP_SIZE': (512, 512),
 'FREEZE': {'MODEL_FILENAME': '__model__',
            'PARAMS_FILENAME': '__params__',
            'SAVE_DIR': 'freeze_model'},
 'MEAN': [0.5, 0.5, 0.5],
 'MODEL': {'BN_MOMENTUM': 0.99,
           'DEEPLAB': {'ASPP_WITH_SEP_CONV': True,
                       'BACKBONE': 'xception_65',
                       'DECODER_USE_SEP_CONV': True,
                       'DEPTH_MULTIPLIER': 1.0,
                       'ENABLE_DECODER': True,
                       'ENCODER_WITH_ASPP': True,
                       'OUTPUT_STRIDE': 16},
           'DEFAULT_EPSILON': 1e-05,
           'DEFAULT_GROUP_NUMBER': 32,
           'DEFAULT_NORM_TYPE': 'bn',
           'FP16': False,
           'HRNET': {'STAGE2': {'NUM_CHANNELS': [40, 80], 'NUM_MODULES': 1},
                     'STAGE3': {'NUM_CHANNELS': [40, 80, 160],
                                'NUM_MODULES': 4},
                     'STAGE4': {'NUM_CHANNELS': [40, 80, 160, 320],
                                'NUM_MODULES': 3}},
           'ICNET': {'DEPTH_MULTIPLIER': 0.5, 'LAYERS': 50},
           'MODEL_NAME': 'unet',
           'MULTI_LOSS_WEIGHT': [1.0],
           'PSPNET': {'DEPTH_MULTIPLIER': 1, 'LAYERS': 50},
           'SCALE_LOSS': 'DYNAMIC',
           'UNET': {'UPSAMPLE_MODE': 'bilinear'}},
 'NUM_TRAINERS': 1,
 'SLIM': {'KNOWLEDGE_DISTILL': False,
          'KNOWLEDGE_DISTILL_IS_TEACHER': False,
          'KNOWLEDGE_DISTILL_TEACHER_MODEL_DIR': '',
          'NAS_ADDRESS': '',
          'NAS_IS_SERVER': True,
          'NAS_PORT': 23333,
          'NAS_SEARCH_STEPS': 100,
          'NAS_SPACE_NAME': '',
          'NAS_START_EVAL_EPOCH': 0,
          'PREPROCESS': False,
          'PRUNE_PARAMS': '',
          'PRUNE_RATIOS': []},
 'SOLVER': {'BEGIN_EPOCH': 1,
            'CROSS_ENTROPY_WEIGHT': None,
            'DECAY_EPOCH': [10, 20],
            'GAMMA': 0.1,
            'LOSS': ['softmax_loss'],
            'LOSS_WEIGHT': {'BCE_LOSS': 1,
                            'DICE_LOSS': 1,
                            'LOVASZ_HINGE_LOSS': 1,
                            'LOVASZ_SOFTMAX_LOSS': 1,
                            'SOFTMAX_LOSS': 1},
            'LR': 0.001,
            'LR_POLICY': 'poly',
            'LR_WARMUP': False,
            'LR_WARMUP_STEPS': 2000,
            'MOMENTUM': 0.9,
            'MOMENTUM2': 0.999,
            'NUM_EPOCHS': 10,
            'OPTIMIZER': 'adam',
            'POWER': 0.9,
            'WEIGHT_DECAY': 4e-05},
 'STD': [0.5, 0.5, 0.5],
 'TEST': {'TEST_MODEL': './saved_model/unet_optic/final'},
 'TRAIN': {'MODEL_SAVE_DIR': './saved_model/unet_optic/',
           'PRETRAINED_MODEL_DIR': './pretrained_model/unet_bn_coco/',
           'RESUME_MODEL_DIR': '',
           'SNAPSHOT_EPOCH': 5,
           'SYNC_BATCH_NORM': False},
 'TRAINER_ID': 0,
 'TRAIN_CROP_SIZE': (512, 512)}
#Device count: 1
W0601 23:32:51.594745 24660 device_context.cc:252] Please NOTE: device: 0, CUDA Capability: 61, Driver API Version: 10.1, Runtime API Version: 10.0
W0601 23:32:51.633760 24660 device_context.cc:260] device: 0, cuDNN Version: 7.3.
load test model: ./saved_model/unet_optic/final

**Comment by xiaowuzilou** [comment]
楼上是命令行界面，没有发现您提到的评估指标
